docs/sdk/dev/vcs.md

```diff
@@ -0,0 +1,19 @@
+---
+title: VCS connector
+description: The VCS connector feature for the Spryker SDK.
```

Comment:
```suggestion
description: VCS connector lets you use a different VCS system with a single command interface.
```
More meaningful meta descriptions would be better. Meta descriptions appear first in the google search results and are useful for users to understand if they should read the doc

Reply:
I'll reach out to the team for further detail on the VCS connector's functions and amend this!

Reply:
you can leave it as I changed it, that's ok :)